### ISSUE TITLE
Question: Referencing images from markdown posts?

### ISSUE THREAD

**Issue by thevangelist** [gatsbyjs/gatsby#2389]
I created a blog with the tutorial on gatsbyjs.org. However, I am unable to reference to images in my markdown - I did some googling and no obvious solution exists.

Here's my current folder structure:

```
├── layouts
│   └── index.js
├── pages
│   ├── 404.js
│   ├── art.js
│   ├── contact.js
│   ├── index.js
│   ├── my-files.js
│   ├── static
│   │   ├── some-image.jpg
│   └── work.js
├── posts
│   ├── 2016-04-04-some-post.md
├── templates
│   └── blog-post.js
└── utils
    └── typography.js
```

TLDR: how do I reference `some-image.jpg` from the single post written in markdown?

**Comment by EmilTholin** [comment]
Maybe looking at [Mathews' blog](https://github.com/KyleAMathews/blog/tree/master/src/pages/2017-1-31---typefaces-easiest-way-to-self-host-fonts) can help?

**Comment by thevangelist** [comment]
Thanks, I was able to solve this (a bug due to migration)

**Comment by snirp** [comment]
@thevangelist It is always, good practice to describe what went wrong, how you found out, and how you resolved it. Just in case somebody else has a bug at the same point and comes here looking.

**Comment by tcg** [comment]
;) Came here looking.

**Comment by jgaehring** [comment]
I think this is what @thevangelist was referring to: https://github.com/gatsbyjs/gatsby-starter-blog/issues/113#issuecomment-408619291

After Migrating to v2 there was a breaking change with was patched by `gatsby@^2.0.0-beta.61` and `gatsby-transformer-remark@^2.1.1-beta.5`.

In my case, I also still needed to install gatsby-remark-images and its dependency, gatsby-plugin-sharp. Hope this helps anyone else who comes looking!

**Comment by 5tormTrooper** [comment]
For anyone that finds this post lacking,
Assuming you have a folder structure like:
```bash
├── pages
│   ├── post-title
│	│	├── index.md
└──	└──	└── image.jpg
```
You can reference the image from inside the post like so:
```md
![](./image.jpg)
```

Make sure you've got [gatsby-transformer-remark](https://www.gatsbyjs.org/packages/gatsby-transformer-remark/?=remark) and [gatsby-remark-images](https://www.gatsbyjs.org/packages/gatsby-remark-images/?=remark) installed and configured.
